PSix/CurrentUser.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  CurrentUser.swift
+//  PSix
+//
+//  Created by Turzion, Avihu on 4/14/15.
+//  Copyright (c) 2015 PayPal. All rights reserved.
+//
+
+import Foundation
+import Parse
+
+class CurrentUser {
```

Comment:
Why is this a class?
You don't have anything saved inside here.
This could defenitly be a struct.